lib/evmone/eof.hpp

```diff
@@ -62,7 +85,20 @@ struct EOF1Header
     /// Offset of every container section start;
     std::vector<uint16_t> container_offsets;
 
-    std::vector<EOFCodeType> types;
+    /// A helper to extract reference to a specific type section.
+    [[nodiscard]] EOFCodeType get_type(bytes_view container, size_t type_idx) const noexcept
+    {
+        static constexpr size_t TYPE_ENTRY_SIZE = 4;  // Size of each type entry in bytes
```

Comment:
```suggestion
        static constexpr size_t TYPE_ENTRY_SIZE = sizeof(EOFCodeType);  // Size of each type entry in bytes
```
Put this constant outside of function and use in other places that now uses `4`. E.g. `type_section_size != code_sizes.size() * 4`.